Description: Use cups-pk-helper in *Mechanism.conf
 .
 cups-pk-helper (0.2.6-1ubuntu4~artful1) artful; urgency=medium
 .
   * Try: prevent delete of other's jobs
Author: Shem Pasamba <shemgp@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-12-20

--- cups-pk-helper-0.2.6.orig/src/org.opensuse.CupsPkHelper.Mechanism.conf
+++ cups-pk-helper-0.2.6/src/org.opensuse.CupsPkHelper.Mechanism.conf
@@ -6,7 +6,7 @@
 <busconfig>
 
   <!-- Only root can own the service -->
-  <policy user="root">
+  <policy user="cups-pk-helper">
     <allow own="org.opensuse.CupsPkHelper.Mechanism"/>
   </policy>
 
